examples/cli/testScenarios.yaml

```diff
@@ -0,0 +1,21 @@
+# input
```

Comment:
There is no documentation for implemented functionality. You should add the information about how to use these tests to the README files and maybe to the Wiki.

Reply:
Will add wiki after merge

Reply:
https://github.com/nirmata/kyverno/wiki/Testing-Policies